sevenseconds/config/vpc.py

```diff
@@ -92,7 +92,11 @@ def configure_vpc(account: AccountData, region, base_ami_id):
     for subnet in subnets:
         configure_subnet(vpc, subnet, account.dry_run, ec2c.get_waiter('subnet_available'))
 
-    nat_instances = create_nat_instances(account, vpc, region)
+    drop_nat = account.config.get("drop_nat", False)
```

Comment:
We shouldn't provision the internal subnet as well in this case (because it won't work anyway). Since it also doesn't actively delete the gateways, let's rename it to something like `enable_internal_subnet`?

Reply:
Updated the PR

Reply:
As discussed previously internal subnets cannot be disabled because RDS dbs may be using them in most accounts.